Jenkinsfile.yml

```diff
@@ -11,7 +11,7 @@ node('ubuntu18.04-OnDemand'){
   stage('build'){
     sh 'mvn clean install'
     print "Archive the artifacts"
-    sh 'cd component-samples && cp -r scripts demo && tar -czvf demo.tar.gz demo && mv demo.tar.gz ..'
+    sh 'cd component-samples && tar -czvf demo.tar.gz demo && mv demo.tar.gz ..'
     archiveArtifacts artifacts: 'demo.tar.gz', fingerprint: true, allowEmptyArchive: false
```

Comment:
Line 37: scripts/keys_gen.sh  to demo/scripts/key_gen.sh

Reply:
++ line 38

Reply:
The scripts folder has been moved from component-samples/scripts to component-samples/demo/scripts, hence the above line isn't required.

Line 37 assumes that the scripts folder is available at component-samples/demo/scripts, hence no change required there.

Reply:
Looks good 👍 .